src/ReinforcementLearningZoo/src/algorithms/policy_gradient/sac.jl

```diff
@@ -179,11 +193,16 @@ function RLBase.update!(p::SACPolicy, batch::NamedTuple{SARTS})
     end
     update!(p.policy, p_grad)
 
+    # Tune entropy automatically
+    if p.automatic_entropy_tuning
+        p.α -= p.lr_alpha * mean(-log_π .- p.target_entropy)
```

Comment:
What's the meaning of  `target_entropy` here? It seems to only be set to `Float32(-action_dims)` at the init step.

Reply:
In original SAC paper:
[![WkqfSJ.png](https://z3.ax1x.com/2021/07/13/WkqfSJ.png)](https://imgtu.com/i/WkqfSJ)
In fact, this loss is linear for α. So I did not use the optimizer.
\bar{\mathcal{H}} is the target entropy, a constant. I initialized it according to the method of the paper:
[![WkLGc9.png](https://z3.ax1x.com/2021/07/13/WkLGc9.png)](https://imgtu.com/i/WkLGc9)

Reply:
Now I see. Thanks for your explanation!

Reply:
![image](https://user-images.githubusercontent.com/49596933/125396425-28a91280-e3ca-11eb-8d12-e9f65fa419cb.png)
In https://arxiv.org/pdf/1812.05905.pdf it implements automatic temperature tuning and the gradient wrt alpha might be the following expression?
I have seen an implementation like that [here](https://github.com/denisyarats/pytorch_sac/blob/81c5b536d3a1c5616b2531e446450df412a064fb/agent/sac.py#L123)

Reply:
Got it

Reply:
Thank you very much. Let me think about it.

Reply:
I think that log \pi and α are irrelevant.